addons/website/static/src/js/widgets/link_popover_widget.js

```diff
@@ -4,6 +4,24 @@ import contentMenu from 'website.contentMenu';
 import weWidgets from 'wysiwyg.widgets';
 import {_t} from 'web.core';
 
+weWidgets.LinkPopoverWidget.include({
+    /**
+     * @override
+     */
+    start: function () {
+        // hide popover while typing on mega menu
+        if (this.$target.parents('.o_mega_menu').length) {
+            this.$target.on('keydown.link_popover', () => {
+                this.$target.popover('hide');
+                clearTimeout(this.inDebounce);
```

Comment:
I would not use a variable saved on `this` (to avoid when not needed and with a name that generic even more :wink:). Just `let timeoutID = undefined;` outside of the event handler assignment then use that variable :+1:

Reply:
Honestly I just checked how this kind of stuff was done in odoo / framework and did it the same way (with the same name also :D ) but I will change it if you prefer

Reply:
Changed